terminus-ui/date-range/testing/src/test-helpers.ts

```diff
@@ -27,18 +94,32 @@ export function createDateRangeGroup(startDate: null | Date = null, endDate: nul
 }
 
 
-export function getDebugRangeInputs(fixture: ComponentFixture<any>): DebugElement[] {
-  return fixture.debugElement.queryAll(By.css('ts-input'));
-}
+/**
+ * Set values for both date range inputs
+ *
+ * @param fixture - The component fixture
+ * @param startDate - The date to set the start input
+ * @param endDate - The date to set the end input
+ * @param index - The select instance
+ * @return The whenStable promise
+ */
+export function setDateRangeValues(fixture: ComponentFixture<any>, startDate: Date, endDate: Date, index = 0): Promise<any> {
```

Comment:
Interesting it returns Promise here... since it's setting value, does it have to have returned type? Or it's because of async stuff?

Reply:
It doesn't _have_ to have a return, but since we can I figured we should. There could be cases where the function is triggering something that isn't instant (many items, animated stuff, timeouts etc) so this way they can chain to the function to make sure they are checking _after_ Angular settles back down:

```typescript
setDateRangeValues(fixture).then(() => {
  expect(...)...
});
```